ccd-definition/ComplexTypes/CareSupervision/ManageDocuments/ManageDocumentType.json

```diff
@@ -18,15 +18,6 @@
     "SecurityClassification": "Public",
     "FieldShowCondition": "type=\"DO NOT SHOW\""
   },
-  {
-    "LiveFrom": "01/01/2017",
-    "ID": "ManageDocumentType",
-    "ListElementCode": "relatedToHearing",
```

Comment:
just doubling checking this complex type is used in transient field only, we are not storing this relatedToHearing in data?

Reply:
this complex type is for transient data captured on 1st page of a flow. Now the fact that document is related to hearing is captured on 2nd page, thus does not belongs to this structure (ccd does not allow you to update same complex object on 2 pages). Case data has manageDocumentsRelatedToHearing which capture this information (it was added there by you  for LA part of story :) )

Reply:
thought so, just making sure :) did not want repeat of previous issue!